themes/login-forms/templates/SilverStripeLogo.ss

```diff
@@ -14,28 +14,26 @@
                 }
 
                 .glyph {
-                    fill: #44C8F5;
+                    fill: #FFFFFF;
```

Comment:
Is this intentional?

Reply:
Hey, I believe this is intentional, I got the svg from one of the designers here in Silverstripe. Because this was a quick fix, therefore I am not sure if this will be the final design. I can double-check with our designers :)